Client/src/components/common/Label/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Label = props => {
+  const { input: Input, id, labelText, labelClassName} = props;
+  return (
+    <label htmlFor={id} className={labelClassName} >
```

Comment:
we want style for the label

Reply:
I said in the PR description that the component that I can't make a style
because of the component used in different places and layout 
so i pass a className as props for the container, label and input 
<label htmlFor={id} className={labelClassName} >